stripe/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -164,7 +164,21 @@ interface StripeIntent : StripeModel {
                 @Parcelize
                 data class Alipay(
                     val data: String
-                ) : MobileData()
+                ) : MobileData() {
+                    /**
+                     * The alipay data string is formatted as query parameters.
+                     * When authenticate is complete, we make a request to the
+                     * return_url, as a hint to the backend to ping Alipay for
+                     * the updated state
+                     */
+                    val authCompleteUrl: String?
+                    get() = runCatching {
```

Comment:
can this be changed to execute at construction?

Reply:
sure